### DISCUSSION TITLE
Add support to Tenant

### DISCUSSION THREAD

**Discussion by lucassmacedo** [rappasoft/laravel-livewire-tables#1066] [Feature Requests]
I'm using this package in a saas application that uses Tenant, **archtechx/tenancy** and when my component it's renderized the logic of tenant are lost.  (where tenant_id = 'xxx').

Actualy i'm rewriting in the component the where:

```
    public function builder(): Builder
    {
        $query = Patient::where('tenant_id', auth()->user()->tenant_id);

        return $query;
    }
```

**Comment by lrljoe** [comment]
archtechx/tenancy presents the current tenant ID as 
```php
tenant('id')
```
Rather than an attribute on the auth()->user()

What're you generating query-wise if you update with that?
